stac_api/runtime/src/api.py

```diff
@@ -0,0 +1,57 @@
+"""FastAPI extensions for the VEDA STAC API."""
+from typing import List
+
+import attr
+
+from stac_fastapi.api.app import StacApi
+from stac_fastapi.api.routes import create_async_endpoint
+
+from .core import VedaCrudClient
+
+
+class VedaStacApi(StacApi):
+    """Veda STAC API."""
+
+    client: VedaCrudClient = attr.ib()
+
+    def register_post_search(self):
+        """Register search endpoint (POST /search).
+        Returns:
+            None
+        """
+        super().register_post_search()
+        self.router.add_api_route(
+            name="Search",
+            path="/collection-search",
```

Comment:
@moradology what do you think about `/collection-search-ids` or something that specifies the endpoint will return a list of IDs? 

I suspect we will eventually also want to implement a `collection-search` that returns an array of collections in the same response format as a `/collections` request. By making this list of IDs endpoint unique, we will not have to coordinate a response format change with users in the future.